odoo/addons/base/models/res_users.py

```diff
@@ -697,6 +697,14 @@ def _search(self, domain, offset=0, limit=None, order=None):
 
     @api.model_create_multi
     def create(self, vals_list):
+        for values in vals_list:
+            if 'groups_id' in values:
+                # complete 'groups_id' with implied groups
+                user = self.new(values)
+                gs = user.groups_id._origin
+                gs = gs | gs.trans_implied_ids
```

Comment:
@Gorash FYI
Two classes are merged here. And these fields will be changed soon.